src/openms/source/APPLICATIONS/ConsoleUtils.cpp

```diff
@@ -121,6 +120,7 @@ namespace OpenMS
         {
           LOG_DEBUG << "output shaping: stty size command failed." << std::endl;
         }
+        pclose(fp);
```

Comment:
PS: also document this behavior and write why we think it is correct and this decision was made